sdk/deviceupdate/azure-deviceupdate/samples/contentfactory.py

```diff
@@ -0,0 +1,104 @@
+from azure.deviceupdate.models import ImportUpdateInput, ImportManifestMetadata, FileImportMetadata
```

Comment:
I think you specified an incorrect license header, at least not one that autorest recognizes. Pretty much all license headers are `MICROSOFT_MIT_NO_VERSION`, so you should switch that in your config file

Reply:
I see that we already have that in the config file.